libs/awscdk/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@winglang/platform-awscdk",
+  "version": "0.0.0",
+  "description": "Wing Platform for AWS CDK",
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org",
+    "tag": "latest"
+  },
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "aws-cdk-lib": "^2.64.0",
+    "constructs": "~10.2.69",
+    "safe-stable-stringify": "^2.4.3",
+    "@winglang/sdk": "workspace:^"
+  },
+  "devDependencies": {
+    "bump-pack": "workspace:^",
+    "typescript": "^4.9.5",
+    "tsup": "^6.7.0",
+    "vitest": "^0.32.4"
+  },
+  "scripts": {
+    "compile": "tsc",
```

Comment:
still trying to get `tsup` set up, had some issues but didnt want to hold up initial reviews while I play with this.